testgrid/config.yaml

```diff
@@ -2843,6 +2843,11 @@ dashboards:
     description: "Uses kubetest to run e2e tests (+Feature:Ingress|NEG) against a cluster created with cluster/kube-up.sh"
     alert_options:
       alert_mail_to_addresses: kubernetes-sig-network-test-failures@googlegroups.com
+  - name: skew-cluster-latest-kubectl-stable1-gce
+    test_group_name: ci-kubernetes-e2e-gce-master-new-gci-kubectl-skew
+    description: stable1 e2e tests run against a master gce cluster using a stable1 kubectl binary
+    alert_options:
+      alert_mail_to_addresses: kubernetes-release-team@googlegroups.com
```

Comment:
is the release team really the right team to own this job? seems like this is sig-cli's responsibility

Reply:
Updated to be sig-cli.